### PR TITLE
[WIP] add sys exit when cmd fails Minor changes

### PR DESCRIPTION
**Pull request by mwaykole**
Signed-off-by: mwaykole <mwaykole@redhat.com>

### CODE REVIEW

File: utils/scripts/ocm/ocm.py [hunk 1]
```diff
@@ -1179,6 +1179,7 @@ def change_cluster_channel_group(self):
         ret = execute_command(run_change_channel_cmd)
         if ret is None:
             log.info("Failed to update the channel to {}".format(self.cluster_name))
+            sys.exit(1)
```
Comment (tarukumar) [inline utils/scripts/ocm/ocm.py:1182]: I think it better if you check for the exiting channel on the cluster before making and modification. If channel is "Candidat" the do not modify it .

Comment (mwaykole) [reply utils/scripts/ocm/ocm.py:1182]: sure  that can be done in different PR this pr is just for sys.exit(1)

Comment (bdattoma) [inline utils/scripts/ocm/ocm.py:1182]: is this funciton going to be used in Robot Framework? maybe tasks. If yes, then we should aboid sys.exit. You could check the solution applied in functions like install_rhoam_addon

**Comment by bdattoma** [comment]
@mwaykole is this PR still necessary? if not, please close it

**Pull request closed without merging**